docs/markdown/snippets/find_library_name.md

```diff
@@ -0,0 +1,3 @@
+## dependencies created by compiler.find_library implement the `name()` method
+
+Which would previously result in Meson crashing.
```

Comment:
It didn't crash though, right? It was just an ordinary "error: no such method"...